### ISSUE TITLE
getLinkItems method from Pivot crashes the web app in IE11

### ISSUE THREAD

**Issue by dinoilic** [microsoft/fluentui#19190]
### Environment Information

- **Package version(s)**:
```
"@fluentui/font-icons-mdl2": "^8.1.1",
"@fluentui/merge-styles": "^8.1.1",
"@fluentui/react": "^8.15.1",
"@fluentui/react-experiments": "^8.1.17",
"@fluentui/react-file-type-icons": "^8.1.1",
"@fluentui/set-version": "^8.1.1",
"@fluentui/style-utilities": "^8.1.1",
"@fluentui/utilities": "^8.1.1",
```
- **Browser and OS versions**: Internet Explorer 11

### Please provide a reproduction of the bug in a codepen:

Unfortunately, I can't provide a repro on this once since I can't reproduce it locally, however it reproduces in SharePoint in Internet Explorer (more information can be given via Teams)

#### Actual behavior:

The application crashes with the following error:

`TypeError: Unable to get property 'linkText' of undefined or null reference`

However, we have not changed anything on our end during the migration from v7 to v8 and we never even used `linkText` in our components.

#### Expected behavior:

The application doesn't crash in IE11

### Priorities and help requested:

Are you willing to submit a PR to fix? **Yes, however there's no known repro or fix at the moment and we need help from Fluent team understanding what has changed in the Pivot component during the migration from v7 to v8** 

Requested priority: **Blocking, we are blocked on deploying Search in SharePoint and Office.com**

Products/sites affected: **SharePoint, Office.com**

**Comment by dinoilic** [comment]
@ling1726 Here's the new issue that's also blocking Search in SharePoint and Office.com from deploying any new versions.

**Comment by ling1726** [comment]
@ecraig12345 we should investigate these IE11 errors, since the v8 docsite currently does not work in IE11

**Comment by dinoilic** [comment]
Also, I talked to @behowell who mentioned he might have an idea what's wrong with this particular one.

**Comment by ecraig12345** [comment]
I suspect the problem is the `isPivotItem` check, because IE 11 doesn't support the `name` property for functions. 

v8:
https://github.com/microsoft/fluentui/blob/a5903f5998355805ed787f677bc070d6bfc5fef3/packages/react/src/components/Pivot/Pivot.base.tsx#L58-L60

v7:
https://github.com/microsoft/fluentui/blob/e851260024a6d8d525a5a171b72c242b3cdf0d5e/packages/office-ui-fabric-react/src/components/Pivot/Pivot.base.tsx#L355-L366

@behowell Will you be able to fix this soon>

**Comment by dinoilic** [comment]
Hi, @ecraig12345 @behowell did you have a chance to take a look at this? We're still in some blocked state where we can't ship code due to this. Let me know if there's anything I can help with.

**Comment by behowell** [comment]
@dinoilic The fix is merged and will be in the next release in the next day or so (the bot will reply to #19238 with version number when it's released).

**Comment by dinoilic** [comment]
Thank you @behowell, @ecraig12345 and @ling1726 for great support with our issues here, thanks for unblocking our work! :)
